types/lib/utils/unresolved.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   lib/utils/unresolved.html
+ */
+
```

Comment:
Maybe do not generate the file if there were no types in it?

Reply:
Currently we mirror the HTML import graph to the reference import graph, so that complicates that a bit, since dependents need to know whether their dependency has anything. See also https://github.com/Polymer/gen-typescript-declarations/blob/master/src/gen-ts.ts#L114